cartography/intel/gcp/compute.py

```diff
@@ -697,6 +718,12 @@ def transform_gcp_subnets(subnet_res: Dict) -> List[Dict]:
         )
         subnet["private_ip_google_access"] = s.get("privateIpGoogleAccess", None)
 
+        default_vpc = get_default_vpc(projectId=projectId, compute=compute)
```

Comment:
do the subnets get created automatically as well? if not, we should not assign predefined / user

Reply:
If the VPC is created in 
- `auto-mode` then the Subnets are created automatically (I have added an extra check for that now on the next line) these will be marked as predefined.
- `custom-mode` then the users have to create the Subnets manually which will be marked as user.